x/distribution/model.go

```diff
@@ -116,8 +116,12 @@ func (b *RevenueBucket) Create(db weave.KVStore, rev *Revenue) (orm.Object, erro
 
 // RevenueAccount returns an account address that is holding funds of a revenue
 // with given ID.
-func RevenueAccount(revenueID []byte) weave.Address {
-	return weave.NewCondition("distribution", "revenue", revenueID).Address()
+func RevenueAccount(revenueID []byte) (weave.Address, error) {
```

Comment:
It is 👍 to check and return an error. We usually do it a bit different with [escrow](https://github.com/iov-one/weave/blob/master/x/escrow/model.go#L90) or [multisig](https://github.com/iov-one/weave/blob/master/x/multisig/context.go#L29) for example where we return the condition rather than the address and let the caller decide on calling validate.